Gordon360/Static Classes/Names.cs

```diff
@@ -82,5 +82,10 @@ public static class Request_Status
         public const string PENDING = "Pending";
         public const string APPROVED = "Approved";
         public const string DENIED = "Denied";
+        private static string[] options = new string[] { PENDING, APPROVED, DENIED };
+        public static string[] GetOptions()
+        {
+            return options;
+        }
```

Comment:
Perhaps a computed property would be cleaner here?
```suggestion
        public static string[] Options => new string[3] { PENDING, APPROVED, DENIED };
```